manifests/m/Microsoft/MaliciousSoftwareRemovalTool/5.134.25060.1001/Microsoft.MaliciousSoftwareRemovalTool.locale.en-US.yaml

```
@@ -8,9 +8,10 @@ Publisher: Microsoft
 PackageName: Malicious Software Removal Tool
 License: Proprietary
 Copyright: © Microsoft Corporation. All rights reserved.
-ShortDescription: Tool to manually run Windows' monthly update checks for malicious domains.
+ShortDescription: Tool to manually run Windows' monthly update checks for malicious softwares.

```

```suggestion
ShortDescription: Tool to manually run Windows' monthly update checks for malicious software.
```
Singular and plural are both `software`